### PR TITLE
Some styling / set up client test environment

### PR DESCRIPTION
**Pull request by jasondstrunk**

### CODE REVIEW

**Comment by jasondstrunk** [comment]
@Bouland95

File: Client/Components/Overview.jsx [hunk 1]
```diff
@@ -3,7 +3,23 @@ import React from 'react';
...
+        <object data="Client/Components/stars.svg" type="image/svg+xml"></object>
+        <div className="break"></div>
+
+        <div className="overview item">VALUE</div>
+        <div className="overview item">STYLE</div>
+        <div className="overview item">SIZING</div>
+        <div className="overview item">COMFORT</div>
+
+    </div>
+    </>
```
Comment (azizbouland) [inline Client/Components/Overview.jsx:22]: Add one before indent before h3
Optional: indent everything between <> and </>

File: Client/Components/SingleReview.jsx [hunk 2]
```diff
@@ -5,13 +5,21 @@ const SingleReview = (props) => {
...
+    <div className="break"></div>
+    <span className="author">{props.element.author}</span>
+    <span className="date">{" - " + moment(props.element.createdAt).startOf('minute').fromNow()}</span>
+    <p className="review-body">{props.element.body}</p>
+    <div className="helpful">{"Helpful: " + props.element.helpfulVotes}</div>
+    <div className="sizing">{"Sizing: " + props.element.sizing}</div>
+    <div className="wrapper" data-anim="base wrapper">
+      <div className="circle" data-anim="base left"></div>
+      <div className="circle" data-anim="base right"></div>
+      TEST
```
Comment (azizbouland) [inline Client/Components/SingleReview.jsx:21]: I think you might've forgotten to delete line 21

**Review by azizbouland — APPROVED** [review]
Couple of formatting comments, otherwise looks good!

**Pull request merged**